runtests.py

```diff
@@ -856,10 +856,7 @@ def run_distutils(self, test_directory, module, workdir, incdir,
                 build_extension.compiler = COMPILER
 
             ext_compile_flags = CFLAGS[:]
-            compiler = COMPILER or sysconfig.get_config_var('CC')
 
-            if self.language == 'c' and 'gcc' in compiler:
-                ext_compile_flags.append('-pedantic')
```

Comment:
Could we restrict this to cygwin, if it matters there? We're not currently requiring C99, so seeing those warnings would be nice.